src/scripts/main.js

```diff
@@ -25,3 +25,21 @@ function createTree(element, data) {
 }
 
 createTree(tree, food);
+function createTree(element, data) {
+  if (!Object.keys(data).length) return;
+
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+    li.textContent = key;
+
+    if (typeof data[key] === 'object') {
+      createTree(li, data[key]);
+    }
```

Comment:
The condition `typeof data[key] === 'object'` is used to check if the value is an object. However, this condition will also be true for arrays and null. Since the data structure is known and controlled, this might not cause issues here, but it's generally better to use `data[key] !== null && typeof data[key] === 'object'` to ensure you're only dealing with non-null objects.